src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -2931,5 +2932,82 @@ void M()
 
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact, WorkItem(32100, "https://github.com/dotnet/roslyn/issues/32100")]
+        public void UsingDeclaration_Flow()
+        {
+            string source = @"
+
+using System;
+class P : IDisposable
+{
+    void M()
+    /*<bind>*/{
+        System.Console.WriteLine(""before"");
```

Comment:
Please remove the before and after printlines, they add a bunch of lines to the diff and aren't really adding any test info.